resources/public/api/conf/2.0/schemas/uk_property_non_fhl_retrieve/response.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
**General comment**: In other endpoints we tend to include the validation for each json variable in the description, e.g. "*The value must be between 0 and 99999999999.99 up to 2 decimal places.*" I can see that in this file we have the "minimum" and "maximum" values listed here but they don't seem to appear in the RAML when previewed online: 
<img width="651" alt="image" src="https://user-images.githubusercontent.com/106095575/180182679-7c43e478-7de3-4796-a9ff-b33a55537ca6.png">